SparseGrids/tsgGridFourier.hpp

```diff
@@ -0,0 +1,117 @@
+
+#ifndef __TASMANIAN_SPARSE_GRID_FOURIER_HPP
+#define __TASMANIAN_SPARSE_GRID_FOURIER_HPP
+
+#include <cstdlib>
+#include <math.h>
+#include <complex>
```

Comment:
* Is it necessary to include the `<complex>` header here? 
* Can this be hidden in the `.cpp` implementation without propagating to the user?

Reply:
When I move it to the `.cpp` and attempt to `make`, I get errors because the functions in the `.hpp` files have inputs of type `std::complex<typename>`.

Reply:
* Yes, the question pertains to whether it is possible to redesign the header so that you avoid including `<complex>`